flagutil/flagutil.go

```diff
@@ -0,0 +1,73 @@
+// Package containing flag utilities. This includes implementations of flag.Value and flag.Getter interfaces. Also
+// contains methods like ValidateFlags.
+package flagutil
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"strings"
+)
+
+// Validates flags and logs a fatal message with all errors if found. Must be called after flag.Parse().
+// This is done by calling Get() on all flag values which implement the flag.Getter interface and collecting errors if returned.
+func ValidateFlags() {
+	var flagErrors []error
+	checkGetError := func(f *flag.Flag) {
+		if fv, ok := f.Value.(flag.Getter); ok {
+			if err, ok := fv.Get().(error); ok && err != nil {
+				flagErrors = append(flagErrors, flagError(fmt.Sprintf("Flag: %s, Error: %s, Usage: %s", f.Name, err.Error(), f.Usage)))
+			}
+		}
```

Comment:
Should this throw some sort of other error if not ok? Or maybe this function should also collect warnings?